src/containers/study/utils/index.js

```diff
@@ -1,20 +1,24 @@
 // @flow
 
-const getParticipantLoginLink = (orgId :UUID, studyId :UUID, participantId :UUID) => {
+const getParticipantLoginLink = (orgId :UUID, studyId :UUID, participantId :string) => {
   const rootUrl = 'https://openlattice.com/chronicle/login';
 
   return `${rootUrl}?organizationId=${orgId}&studyId=${studyId}&participantId=${participantId}`;
 };
 
-const getTimeUseDiaryLink = (orgId :UUID, studyId :UUID, participantId :UUID) => {
+const getTimeUseDiaryLink = (orgId :UUID, studyId :UUID, participantId :string) => {
   const rootUrl = 'https://openlattice.com/chronicle/#/time-use-diary';
 
   return `${rootUrl}?organizationId=${orgId}&studyId=${studyId}&participantId=${participantId}`;
 };
 
-/* eslint-disable import/prefer-default-export */
+const getAppUsageLink = (orgId :UUID, studyId :UUID, participantId :string) => {
+  const rootUrl = 'https://openlattice.com/chronicle/#/survey';
```

Comment:
This shouldn't be hardcoded against production. `window.location` has everything you need to construct the URL and I believe `window.location.href` gives you everything up to the `#`.